lib/modules/showImages.js

```diff
@@ -260,6 +260,11 @@ module.options = {
 		value: true,
 		description: 'Autoplay inline videos',
 	},
+	keepNativeExpando: {
```

Comment:
I'm not sure that we need an option for this, since we've been doing it for a while without one (unless you think you'd use it)